src/assets/styles/explore-opps.css

```diff
@@ -0,0 +1,64 @@
+/*
+Explore-opps form overrides
+=============================================== */
+.explore-opps-item {
+    display: flex;
+    justify-content: space-between;
+  }
+  
+  .explore-opps-header {
+    display: flex;
+    justify-content: space-around;
+    padding-top: 8px;
+  }
+
+  .explore-opps-title {
+    width: 50%;
+    text-align: center;
+  }
+  
+  .explore-opps-form {
+    width: 50%;
+  }
+  
+  div.form-group.explore-opps-formgroup >
+  .baseline-value-label,
+  form >
+  div >
+  .baseline-value-label {
+    width: 100%;
+    padding-bottom: 4px;
+  }
+  
+  .baseline-value-input {
+    font-size: .75em;
+  }
+  
+  .form-group.explore-opps-formgroup {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    text-align: center;
+  }
+  
+  div.form-group.explore-opps-formgroup >
+  label.mod-value-label {
+    width: 100%;
+    padding-bottom: 4px;
+    align-self: center;
+  }
+  
+  div.form-group.explore-opps-formgroup >
+  select.mod-value-input {
+    width: 100%;
+  }
+
+  hr.form-break-rule {
```

Comment:
probably don't need to make custom class here, bootstrap has `mt-0` for `margin-top: 0`, `w-100` for `width: 100%`